test/api/core/hotelClerk/addToChannels.test.js

```diff
@@ -1,4 +1,5 @@
-var
+require('reify');
```

Comment:
It allows NodeJS to correctly interpret ES6 class import commands (`import ... from ...`).
Normally there is only 1 instance of reify required at the start of Kuzzle, but I found while working on this PR that some unit tests files also need it to allow running them independantly.